### PR TITLE
Fixed bug when there are multiple root nodes

### PR DESCRIPTION
**Pull request by FarisElghlan**
Also changed the drop event to not set the completed status to true,
so when dragging from a text editor the text will remain there.

Now the graph of 328 genomes can be drawn without bugs. Next todo: speed up the drawing of edges.

### CODE REVIEW

File: PL2/PL2-core/src/main/java/nl/tudelft/pl2016gr2/core/algorithms/subgraph/SubgraphAlgorithmManager.java [hunk 1]
```diff
@@ -21,6 +21,8 @@
  */
 public class SubgraphAlgorithmManager {
 
+  private static int dummyRootNodeId = Integer.MAX_VALUE;
```
Comment (wouterjsmit) [inline PL2/PL2-core/src/main/java/nl/tudelft/pl2016gr2/core/algorithms/subgraph/SubgraphAlgorithmManager.java:24]: Aren't we running out of Integer ranges to assign unique IDs?

**Comment by wouterjsmit** [comment]
Do we not know why the root nodes bug? It seems very fragile to just put a dummy node in front of it.

File: PL2/PL2-core/src/main/java/nl/tudelft/pl2016gr2/core/algorithms/subgraph/SubgraphAlgorithmManager.java (hunk 1 above)
Comment (FarisElghlan) [inline PL2/PL2-core/src/main/java/nl/tudelft/pl2016gr2/core/algorithms/subgraph/SubgraphAlgorithmManager.java:24]: We can still use `Integer.MAX_VALUE / 2 ++`, `Integer.MAX_VALUE / 2 -1 --` and the same for `MIN_VALUE` :)

**Comment by wouterjsmit** [comment]
Right, quick fix is quick

**Pull request merged**